tests/js/tstdlib_imports.nim

```diff
@@ -64,4 +64,7 @@ import
   # works but shouldn't: segfaults
 
```

Comment:
maybe a TODO for future PR so feel free to ignore but I feel the best here would be a glob approach, similar to what I did in https://github.com/nim-lang/Nim/pull/13221 ; this could work  by adding a test inside tests/trunner.nim that would populate the list of all modules (via glob), and use a blacklist for things that don't work (instead of what you're doing which is a whitelist).

The point of a blacklist (instead of whitelist) is to keep working even if modules are added, removed, or moved.

Reply:
To add onto this: I was thinking modules that do not support JS would `include "jsincompatible"` or whatever similar to `include "system/inclrtl"` which would error on JS with an error message. The tester could check for that specific error message, fail if a module not in the blacklist gives it, fail if a module in the blacklist doesn't give it.